src/cppcache/src/ExecutionImpl.cpp

```diff
@@ -100,33 +94,31 @@ ResultCollectorPtr ExecutionImpl::execute(const char* fn, uint32_t timeout,
   bool serverIsHA = false;
   bool serverOptimizeForWrite = false;
 
-  if (verifyFuncArgs) {
-    std::vector<int8_t>* attr = getFunctionAttributes(fn);
-    {
+  std::vector<int8_t>* attr = getFunctionAttributes(fn);
+  {
```

Comment:
What is the purpose of this curly-brace block? Is there an RAII instance in there?